### ISSUE TITLE
`prefer-presence-queries` to report `.not.toEqual(null)` and `.not.toStrictEqual(null)`

### ISSUE THREAD

**Issue by zaicevas** [testing-library/eslint-plugin-testing-library#517]
### What rule do you want to change?

prefer-presence-queries

### Does this change cause the rule to produce more or fewer warnings?

More warnings

### How will the change be implemented?

`prefer-presence-queries` would cover `queryBy*` + `.not.toEqual(null)`/`.not.toStrictEqual(null)` when presence `getBy*` queries can be used.

### Example code

```
      // BAD
      expect(screen.queryByText('Women')).not.toEqual(null)
      expect(screen.queryByText('Women')).not.toStrictEqual(null)

      // GOOD
      expect(screen.getByText('Women')).toBeInTheDocument();
      expect(screen.getByText('Women')).not.toBeNull();
      expect(screen.getByText('Women')).not.toBeFalsy();
```

### How does the current rule affect the code?

Cases under `BAD` are not reported.

### How will the new rule affect the code?

It would report two new cases.

### Do you want to submit a pull request to change the rule?

No
